helm/loki/values.yaml

```diff
@@ -93,6 +95,7 @@ loki:
   loki:
     image:
       repository: giantswarm/loki
+      tag: ""
```

Comment:
Why is this tag empty ?
Where is the actual tag for the given image ?

Reply:
I hesitated to put a comment there... here we're removing the default tag that could be set in the loki chart, like here: https://github.com/grafana/loki/blob/helm-loki-4.4.2/production/helm/loki/values.yaml#L50
This way we're more confident with the used image.
Like with the `_helpers` stuff, it's a workaround, may not be needed in future versions, but won't hurt to keep it IMO.

Reply:
I would rather suggest to put the default value in there, just to be explicit.

Reply:
I prefer not, because that would be one more place to update when upgrading.
I added a comment though, explaining what is used then.
Are you ok with that?

Reply:
As herve said, I'd rather we do not had places we need to change whenever we upgrade

Reply:
In most chart this defautls to the AppVersion in the chart.yaml anyway

Reply:
Ok fine for me